homeassistant/components/tractive/entity.py

```diff
@@ -25,6 +30,24 @@ def __init__(
             sw_version=tracker_details["fw_version"],
             model=tracker_details["model_number"],
         )
-        self._user_id = user_id
+        self._user_id = client.user_id
         self._tracker_id = tracker_details["_id"]
         self._trackable = trackable
+        self._client = client
+
+    async def async_added_to_hass(self) -> None:
+        """Handle entity which will be added."""
+        if not self._client.subscribed:
+            self._client.subscribe()
+
```

Comment:
I would place these two methods here. So they cannot be forgotten.

```suggestion
	 self.async_on_remove(
            async_dispatcher_connect(
                self.hass,
                f"{TRACKER_POSITION_UPDATED}-{self._tracker_id}",
                self._handle_position_update,
            )
	)
	
	 self.async_on_remove(
            async_dispatcher_connect(
                self.hass,
                f"{SERVER_UNAVAILABLE}-{self._user_id}",
                self.handle_server_unavailable,
            )
        )
```

Reply:
If we move those two to the `TractiveEntity`, the TractiveDeviceTracker won't be able to call `super().async_added_to_hass()` beacause device tracker entity doesn't use entity description and has different methods to handle events.